person.rb

```diff
@@ -0,0 +1,44 @@
+require './nameable'
+require './capitalize_decorator'
+require './trimmer_decorator'
+
+class Person < Nameable
+  def initialize(name = 'unknown', age, parent_permission: true) # rubocop:todo Style/OptionalArguments
+    super()
+    @id = Random.rand(1..100)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+    @rentals = []
+  end
+
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def can_use_service?
+    is_of_age? || @parent_permission
+  end
+
+  def correct_name
+    @name
+  end
+
+  def add_rental(rental)
+    @rental << rental
```

Comment:
[Optional]

- [ ] Kindly fix the typo here, it's supposed to be `@retals` instead of @rental 💯